docs/monitors/windows_event_log_monitor.md

```diff
@@ -159,6 +159,7 @@ Log into DataSet and query [monitor = 'windows_process_metrics'](https://app.sca
 | `remote_password`            | Optional (defaults to `none`). Password to use for authentication on the remote server.  This option is only valid on Windows Vista and above. | 
 | `remote_domain`              | Optional (defaults to `none`). The domain for the remote user account. This option is only valid on Windows Vista and above. | 
 | `json`                       | Optional (defaults to `false`). Format events as json? Supports inclusion of all event fields. This option is only valid on Windows Vista and above. | 
+| `placeholder_render`         | Optional (defaults to `false`). Render %%n placeholders in event data?  This option is only valid on Windows Vista and above. | 
```

Comment:
Do we support older than Vista ? If we do, what happens if this is set to `true` on older versions?  Do we silently ignore?

Reply:
Yes, for context there are two Windows apis for retrieving event logs, one is pre-Vista (its use in the monitor code is encapsulated by the OldApi class) and one is Vista & onwards (NewApi & NewJsonApi classes).

Yes, if json or placeholder_render are set as true for pre-Vista it is silently ignored.